packages/next/src/server/base-server.ts

```diff
@@ -2514,7 +2523,7 @@ export default abstract class Server<
         isOnDemandRevalidate,
         isDraftMode: isPreviewMode,
         isServerAction,
-        postponed,
+        postponed: skipPostponed ? undefined : postponed,
```

Comment:
I don't understand how this is enough. In non minimal mode something sends the partial result. Where is that code? and how are we preventing that partial result from being sent before we kick off the now fully dynamic render